tests/yield_from_test.py

```diff
@@ -184,6 +184,48 @@ def test_fix_async_yield_from(s, expected):
         '        yield x\n'
         '    else:\n'
         '        print("boom!")\n',
+        pytest.param(
+            'for x in z:\n'
+            '    yield x\n',
+            id='yield outside function',
+        ),
+        pytest.param(
+            'lambda: [(yield x) for x in z]',
```

Comment:
I don't think this test is relevant, there's no `for` loop here

Reply:
Addressed in 9572c5a7dadf6a4f8c5877462aa23d05be9168fc.